src/server/system_services/bucket_server.js

```diff
@@ -1054,9 +1088,9 @@ function _inject_usage_to_cloud_bucket(target_name, endpoint, usage_list) {
 // UTILS //////////////////////////////////////////////////////////
 
 
-function find_bucket(req, bucket_name = req.rpc_params.name) {
+function find_bucket(req, { bucket_name = req.rpc_params.name, include_deleting } = {}) {
```

Comment:
the original bucket name was changed, is the include_deleting really needed? and if so, shouldn't we look for the new name and not the original?

Reply:
I think you are right. I'll change it and test

Reply:
@jackyalbo I made the changes, can you review again? thanks